docs/operations/common_configuration/zone_aware_routing.rst

```diff
@@ -0,0 +1,13 @@
+.. _common_configuration_zone_aware_routing:
+
+Zone aware routing
```

Comment:
Should we add a way for them to validate it is working? Example stats to look at?